DigitalLearningSolutions.Web/Views/Support/SupportTickets.cshtml

```diff
@@ -0,0 +1,40 @@
+﻿@using DigitalLearningSolutions.Web.Models.Enums
+@using DigitalLearningSolutions.Web.ViewModels.Support
+@model SupportTicketsViewModel
+
+<link rel="stylesheet" href="@Url.Content("~/css/support/support.css")" asp-append-version="true">
+
+@{
+  ViewData["Title"] = "Support tickets";
+  ViewData["Application"] = Model.DlsSubApplication.HeaderExtension;
+  ViewData["HeaderPath"] = Model.DlsSubApplication.HeaderPath;
+  ViewData["HeaderPathName"] = Model.DlsSubApplication.HeaderPathName;

```

Comment:
We shouldn't need these, you have the SetDlsSubApplication attribute on the controller

Reply:
Done. I've removed them from the original support view also.